db/seeds/index.js

```diff
@@ -1,10 +1,15 @@
 'use strict';
 
-const Movies = require('./data/movies');
+const Locations       = require('./data/locations');
+const LocationsMovies = require('./data/locations-movies');
+const Movies          = require('./data/movies');
 
 exports.seed = (Knex) => {
-  return Knex('movies').truncate()
+  return Knex('movies').insert(Movies)
```

Comment:
nit: you can actually insert Movies and Locations at the same time since they're not dependent on each other! 
here's an example on how to do that
https://github.com/lob/lob-api/blob/99a95ec5e191c9f04e9344d2fe6282865d580c80/test/plugins/features/credits/controller.test.js#L26-L30